.github/workflows/core-plugins-build-and-release.yml

```diff
@@ -14,6 +14,7 @@ on:
           - parca
           - stackdriver
           - tempo
+          # - zipkin
```

Comment:
fwiw you can uncomment this now (and let me know if you find any issue with the release)

Reply:
Yeah I had you tagged [here](https://github.com/grafana/grafana/pull/83116) to check this in fact!